src/modules/0.0.16/iasql_platform/index.ts

```diff
@@ -1,14 +1,14 @@
 /* THIS MODULE IS A SPECIAL SNOWFLAKE. DON'T LOOK AT IT FOR HOW TO WRITE A REAL MODULE */
 
-import { Module2, } from '../../interfaces'
+import { ModuleBase, } from '../../interfaces'
 import * as metadata from './module.json'
 import { IasqlModule, IasqlTables, } from './entity'
 
-export const IasqlPlatform: Module2 = new Module2({
-  ...metadata,
-  utils: {
-    IasqlModule,
-    IasqlTables,
-  },
-  mappers: {},
-}, __dirname);
\ No newline at end of file
+class IasqlPlatform extends ModuleBase {
+  constructor() { super(); super.init(); }
+  dirname = __dirname;
+  dependencies = metadata.dependencies;
+  IasqlModule = IasqlModule;
+  IasqlTables = IasqlTables;
```

Comment:
any reason to have these 2 capitalized? I think lines like https://github.com/iasql/iasql-engine/pull/1068/files#diff-c8f7acf4155d692e9663c0ae07f0ab80ceb39ab0c6d7e15d9a2e987aa1a04ac9R806 read weird

Reply:
I was just trying to keep the difference between the versions minimized, but I guess with all of the conditionals in there anyways, I don't have to.

I can adjust them.